src/main/java/com/exadel/sandbox/em_notif/contoller/EmployeeNotificationController.java

```diff
@@ -0,0 +1,52 @@
+package com.exadel.sandbox.em_notif.contoller;
+
+import com.exadel.sandbox.em_notif.dto.EmployeeNotificationBaseDto;
+import com.exadel.sandbox.em_notif.dto.EmployeeNotificationResponseDto;
+import com.exadel.sandbox.em_notif.repository.EmplyeeNotificationRepository;
+import com.exadel.sandbox.em_notif.service.EmployeeNotificationService;
+import com.exadel.sandbox.notification.dto.NotificationBaseDto;
+import com.exadel.sandbox.notification.dto.NotificationResponseDto;
+import com.exadel.sandbox.notification.service.NotificationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("employee_notification")
+@RequiredArgsConstructor
+public class EmployeeNotificationController{
+    private EmployeeNotificationService employeeNotificationService;
+
+    @GetMapping("list")
+    List<EmployeeNotificationResponseDto> getEmplyeeNotifications() {
+        return employeeNotificationService.getEmployeeNotifications();
+    }
+
+    @GetMapping("{id}")
+    EmployeeNotificationResponseDto getById(@PathVariable("id") Long id) {
+
+        return employeeNotificationService.getEmNotifById(id);
+    }
+
+    @PostMapping()
+    EmployeeNotificationResponseDto createEmployeeNotification(@RequestBody EmployeeNotificationBaseDto employeeNotificationBaseDto) {
```

Comment:
U need 4 DTOs
1. ResponseDTO (with not updating fields)
2. CreateDTO with fields for create
3. Update DTO with fields for update
4.BaseDTO to take out common fields